docs/how-to/extract-a-backup.md

```diff
@@ -145,3 +144,29 @@ When you're all done exploring your files, unmount your mount point. No
 ```bash
 borgmatic umount --mount-point /mnt
 ```
+
+## Extract the configuration files used to create an archive
+
+<span class="minilink minilink-addedin">New in version 1.7.15</span> borgmatic
+automatically stores all the configuration files used to create an archive inside the
+archive itself. This is useful in cases where you've lost a configuration
+file, or you want to see what configurations were used to create a particular
+archive.
+
+To extract the configuration files from an archive, use the `config bootstrap` action. For example:
+
+```bash 
+borgmatic config bootstrap --repository repo.borg --destination /tmp
+```
+
+This extracts the configuration file from the latest archive in the repository `repo.borg` to `/tmp/etc/borgmatic/config.yaml`, assuming that the only configuration file used to create this archive was located at `/etc/borgmatic/config.yaml` when the archive was created.
+
+Note that to run the `config bootstrap` action, you don't need to have a borgmatic configuration file. You only need to specify the repository to use via the `--repository` flag, borgmatic will figure out the rest.
+
+If a destination directory is not specified, the configuration files will be extracted to their original locations, silently **overwriting** any configuration files that may already exist. For example, if a configuration file was located at `/etc/borgmatic/config.yaml` when the archive was created, it will be extracted to `/etc/borgmatic/config.yaml` too. This is in line with the `extract` action, which extracts files to their original locations by default.
```

Comment:
```suggestion
If a destination directory is not specified, the configuration files will be extracted to their original locations, silently **overwriting** any configuration files that may already exist. For example, if a configuration file was located at `/etc/borgmatic/config.yaml` when the archive was created, it will be extracted to `/etc/borgmatic/config.yaml` too.
```